Report/Chapters/Testplan.tex

```diff
@@ -1,7 +1,21 @@
 \chapter{Test Plan}
 
 To test the finished product, we used (automatic) unit tests as well as (manual) acceptance tests.
+The acceptance tests are focsued on the front end as it is difficult to use automated unit tests for this purpose.
```

Comment:
> focsued

Typo

Aside from that, you should turn the sentence around:
Since unit tests are hard for front end we used acceptance tests.